week-9/Homework/mandatory/2-exercises/1-shopping-cart.js

```diff
@@ -10,17 +10,19 @@ The output of running your code should be:
 */
 
 class ShoppingCart {
-  // Add your code here
-
-  cartContains() {
-    // Use console.log() to output everything contained in your cart
-  }
+	constructor(toiletRoll, eggs, pasta) {
+		this.toiletRoll = toiletRoll;
+		this.eggs = eggs;
+		this.pasta = pasta;
```

Comment:
This is also a possible solution. But the problem is that in this way you cast in stone the number of elements the cart can store. Also, each of them has a name that indicates what should be there. The shopping cart should be a generic cart, it should be able to store anything. In this exercise, Shopping cart needs an array to where you will be able to add the items using the `addItem` function that should be also implemented.